contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -296,9 +319,8 @@ contract NodeOperatorsRegistry is AragonApp, IStakingModule, Versioned {
     function setNodeOperatorName(uint256 _nodeOperatorId, string _name) external {
         _onlyValidNodeOperatorName(_name);
         _onlyExistedNodeOperator(_nodeOperatorId);
-        _authP(SET_NODE_OPERATOR_NAME_ROLE, arr(uint256(_nodeOperatorId)));
+        _authP(ADD_NODE_OPERATOR_ROLE, arr(uint256(_nodeOperatorId)));
 
-        require(keccak256(_nodeOperators[_nodeOperatorId].name) != keccak256(_name), "NODE_OPERATOR_NAME_IS_THE_SAME");
```

Comment:
let's revert this change since we have some space for that

Reply:
+